main/main.c

```diff
@@ -77,9 +80,14 @@ void print_buffer(uint8_t* buffer, uint8_t size, uint8_t address) {
 	LOG("+----+-------------+---------+");
 }
 
-esp_err_t on_wifi_connect(void) {
-	LOG("Yay we connected!");
-	return ESP_OK;
+void on_wifi_connect(void) {
+	connection_data_t connection_data;
+	if (persistent_storage_get_connection_data(&connection_data) != ESP_OK) {
+		// TODO: Enter sleep mode
+	}
+
+	uint32_t sensor_values[] = {2, 69, 420, 1, 2, 3, 4};
```

Comment:
Behalve dat we maar 3 sensoren hebben, prima. Heb je uitgezocht hoe de data nu naar de server wordt gestuurd. Volgens mij zit daar ook de naam van de sensor bij, zodat je weet van welke sensor de data komt.

Reply:
The name of the sensor is incorporated in the URL we send the request too
The fields include values other than the pure sensors as well, like firmware version

Although some of these fields are redundant yes, but it can easily be modified in a later PR